src/specs/specs-idl.json

```diff
@@ -13,7 +13,9 @@
     "https://immersive-web.github.io/webxr-gamepads-module/",
     "https://notifications.spec.whatwg.org/",
     "https://storage.spec.whatwg.org/",
+    "https://svgwg.org/specs/animations/",
     "https://url.spec.whatwg.org/",
+    "https://w3c.github.io/gamepad/extensions.html",
```

Comment:
Another shortname generation issue here: this will generate `gamepad` for the IDL file (and clash with the IDL export of the actual Gamepad spec), instead of the forecasted `gamepad-extensions` shortname.

Reffy actually does not extract any WebIDL from this spec in practice, because it does not yet have a "conformance" section and thus gets flagged by Respec as "informative only" (see w3c/respec#2580). This should rather be fixed in the Gamepad extensions spec.

Reply:
I've sent https://github.com/w3c/gamepad/pull/125, will omit this until that's fixed.